### PR TITLE
Add Crust into dStorage section

### PR DESCRIPTION
**Pull request by badkk**

### CODE REVIEW

File: docs/develop/crusthelpers.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - IPFS
+  - matic
+image: https://matic.network/banners/matic-network-16x9.png
+---
+
+[Crust](https://crust.network) provides a decentralized storage network for Web3. It is designed to realize core values of decentralization, privacy and assurance. Crust supports multiple storage-layer protocols such as IPFS and exposes instant accessible on-chain storage functions to users.
+
+The Crust protocol is 100% compatible with the [IPFS](https://ipfs.io/) protocol, and it matches people who have hard drive space to spare with those who need to store data or host content. Crust supports most contract platforms, such as Polygon/Solana/NEAR/etc. with its cross-chain solution.
+
+> To learn more about Crust, check its [Decentralized Storage market](https://wiki.crust.network/docs/en/DSM) and [Cross-chain Storage Solution](https://wiki.crust.network/docs/en/buildCrossChainSolution#ii-native-ipfs-integration). Also, you can start building with [Crust Build-101](https://wiki.crust.network/docs/en/build101)
```
Comment (DannyS03) [inline docs/develop/crusthelpers.md:18]:
```suggestion
:::info Learn more about Crust
Check out the [Decentralized Storage market](https://wiki.crust.network/docs/en/DSM) and [Cross-chain Storage Solution](https://wiki.crust.network/docs/en/buildCrossChainSolution#ii-native-ipfs-integration) documentation. You can also start building with Crust Storage by following the [Crust Build-101](https://wiki.crust.network/docs/en/build101) guide.
:::
```

File: docs/develop/crusthelpers.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+keywords:
+  - Crust Network
+  - crustfiles
+  - crust-polygon bridge
+  - IPFS
+  - matic
+image: https://matic.network/banners/matic-network-16x9.png
+---
+
+[Crust](https://crust.network) provides a decentralized storage network for Web3. It is designed to realize core values of decentralization, privacy and assurance. Crust supports multiple storage-layer protocols such as IPFS and exposes instant accessible on-chain storage functions to users.
```
Comment (DannyS03) [inline docs/develop/crusthelpers.md:14]:
```suggestion
[Crust Network](https://crust.network) provides a decentralized storage network for web3. It is designed to realize the core values of decentralization, privacy and assurance. Crust supports multiple storage-layer protocols such as IPFS, and exposes instantly accessible on-chain storage functions to users.
```

File: docs/develop/crusthelpers.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+The Crust protocol is 100% compatible with the [IPFS](https://ipfs.io/) protocol, and it matches people who have hard drive space to spare with those who need to store data or host content. Crust supports most contract platforms, such as Polygon/Solana/NEAR/etc. with its cross-chain solution.
```
Comment (DannyS03) [inline docs/develop/crusthelpers.md:16]:
```suggestion
The protocol matches people who have hard drive space to spare with those who need to store data or host content. Crust supports most smart contract platforms with its cross-chain solution, including Polygon. Find out all the platforms that Crust storage supports on their [official website](https://crust.network/).
```

File: docs/develop/crusthelpers.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+> To learn more about Crust, check its [Decentralized Storage market](https://wiki.crust.network/docs/en/DSM) and [Cross-chain Storage Solution](https://wiki.crust.network/docs/en/buildCrossChainSolution#ii-native-ipfs-integration). Also, you can start building with [Crust Build-101](https://wiki.crust.network/docs/en/build101)
+
+## Crust Storage Helpers
+
+- [Crust Files](https://crustfiles.io): the 1st web3 personal cloud storage build by Crust community, supports multi-web3 wallet login and provide fully decentralized and privacy storage function for all web3 users.
```
Comment (DannyS03) [inline docs/develop/crusthelpers.md:22]:
```suggestion
- [Crust Files](https://crustfiles.io): Built by the Crust community, Crust Files is first web3 personal cloud storage that supports multi-web3 wallet login and provides fully decentralized and privacy storage function for all web3 users.
```

File: docs/develop/crusthelpers.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+- [IPFS W3Auth Gateway](https://docs.ipfs.tech/concepts/ipfs-gateway/#authenticated-gateways): Web3 authenticated IPFS gateway, support all the IPFS *WRITE* functions, that helps users upload file or host content after pass the web3 auth verification.
```
Comment (DannyS03) [inline docs/develop/crusthelpers.md:23]:
```suggestion
- [IPFS W3Auth Gateway](https://docs.ipfs.tech/concepts/ipfs-gateway/#authenticated-gateways): A web3 authenticated IPFS gateway that supports all the IPFS *WRITE* functions and helps users upload files or host content after passing the W3Auth verification.
```

File: docs/develop/crusthelpers.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+- [IPFS W3Auth Pinning Service](https://wiki.crust.network/docs/en/buildIPFSW3AuthPin): A standard IPFS pinning service with web3 authentication, it allows users access Crust storage resources without interacting with Crust blockchain.
```
Comment (DannyS03) [inline docs/develop/crusthelpers.md:24]:
```suggestion
- [IPFS W3Auth Pinning Service](https://wiki.crust.network/docs/en/buildIPFSW3AuthPin): A standard IPFS pinning service with web3 authentication that allows users to access Crust storage resources without needing to interact with the Crust blockchain.
```

File: docs/develop/crusthelpers.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+- [Crust Github Action](https://github.com/marketplace/actions/crust-ipfs-pin): A standard github action which can involve the fully decentralized hosting CI/CD with Crust.
```
Comment (DannyS03) [inline docs/develop/crusthelpers.md:25]:
```suggestion
- [Crust Github Action](https://github.com/marketplace/actions/crust-ipfs-pin): A standard GitHub action which can include fully decentralized CI/CD hosting with Crust.
```

File: docs/home/blockchain-basics/basics-consensus-mechanism.md [hunk 8]
```diff
@@ -23,7 +23,7 @@ Proof-of-Stake mechanism achieve consensus by requiring users to stake an amount
 This form of consensus mirrors the election of members in governing bodies. Witnesses, those who validate transactions, are elected.
 
 4. PoSpace: Proof of Space <br></br>
-This kind of consensus mechanism is useful in decentralized file storage applications like storj.io, filecoin, where nodes prove they have legitimate capacity in their hardware. Sometimes also referred as PoStorage or PoCapacity.
+This kind of consensus mechanism is useful in decentralized file storage applications like storj.io, filecoin, crust, where nodes prove they have legitimate capacity in their hardware. Sometimes also referred as PoStorage or PoCapacity.
```
Comment (DannyS03) [inline docs/home/blockchain-basics/basics-consensus-mechanism.md:26]:
```suggestion
This kind of consensus mechanism is useful in decentralized file storage applications like in Storj.io, Filecoin, and Crust, where nodes prove they have legitimate capacity in their hardware. Sometimes also referred as PoStorage or PoCapacity.
```

**Review by DannyS03 — CHANGES_REQUESTED** [review]
thank you for the contribution @badkk! left some suggestions. Some of the other pages will likely be updated soon too

**Comment by badkk** [comment]
> thank you for the contribution @badkk! left some suggestions. Some of the other pages will likely be updated soon too

Thank you so much for your comments @DannyS03 ! I've already changed and updated, please take a look!

File: docs/develop/crusthelpers.md [hunk 9]
```diff
@@ -11,16 +11,17 @@ keywords:
...
+The protocol matches people who have hard drive space to spare with those who need to store data or host content. Crust supports most smart contract platforms with its cross-chain solution, including Polygon. Find out all the platforms that Crust storage supports on their [official website](https://crust.network/).
 
-> To learn more about Crust, check its [Decentralized Storage market](https://wiki.crust.network/docs/en/DSM) and [Cross-chain Storage Solution](https://wiki.crust.network/docs/en/buildCrossChainSolution#ii-native-ipfs-integration). Also, you can start building with [Crust Build-101](https://wiki.crust.network/docs/en/build101)
+:::info Learn more about Crust
+Check its [Decentralized Storage market](https://wiki.crust.network/docs/en/DSM) and [Cross-chain Storage Solution](https://wiki.crust.network/docs/en/buildCrossChainSolution#ii-native-ipfs-integration). Also, you can start building with [Crust Build-101](https://wiki.crust.network/docs/en/build101)
```
Comment (DannyS03) [inline docs/develop/crusthelpers.md:22]:
```suggestion
:::
```

File: docs/develop/crusthelpers.md [hunk 10]
```diff
@@ -11,16 +11,17 @@ keywords:
...
```
Comment (DannyS03) [inline docs/develop/crusthelpers.md]:
```suggestion
Check out the [Decentralized Storage market](https://wiki.crust.network/docs/en/DSM) and [Cross-chain Storage Solution](https://wiki.crust.network/docs/en/buildCrossChainSolution#ii-native-ipfs-integration) offered by Crust Network. Also, you can start building with [Crust Build-101](https://wiki.crust.network/docs/en/build101).
```

File: docs/develop/crusthelpers.md [hunk 11]
```diff
@@ -11,16 +11,17 @@ keywords:
...
 
 ## Crust Storage Helpers
 
-- [Crust Files](https://crustfiles.io): the 1st web3 personal cloud storage build by Crust community, supports multi-web3 wallet login and provide fully decentralized and privacy storage function for all web3 users.
-- [IPFS W3Auth Gateway](https://docs.ipfs.tech/concepts/ipfs-gateway/#authenticated-gateways): Web3 authenticated IPFS gateway, support all the IPFS *WRITE* functions, that helps users upload file or host content after pass the web3 auth verification.
-- [IPFS W3Auth Pinning Service](https://wiki.crust.network/docs/en/buildIPFSW3AuthPin): A standard IPFS pinning service with web3 authentication, it allows users access Crust storage resources without interacting with Crust blockchain.
-- [Crust Github Action](https://github.com/marketplace/actions/crust-ipfs-pin): A standard github action which can involve the fully decentralized hosting CI/CD with Crust.
+- [Crust Files](https://crustfiles.io): Built by the Crust community, Crust Files is first web3 personal cloud storage that supports multi-web3 wallet login and provides fully decentralized and privacy storage function for all web3 users.
```
Comment (DannyS03) [inline docs/develop/crusthelpers.md:25]:
```suggestion
- [Crust Files](https://crustfiles.io): Built by the Crust community, Crust Files is the first personal cloud storage for web3 that supports multi-web3 wallet login and provides fully decentralized and privacy storage function for all users.
```

**Review by DannyS03 — CHANGES_REQUESTED** [review]
thanks @badkk! Just a few more minor suggestions

**Comment by badkk** [comment]
> thanks @badkk! Just a few more minor suggestions

Sure! @DannyS03 Already solved!

**Comment by DannyS03** [comment]
cheers @badkk

**Review by DannyS03 — APPROVED** [review]

**Pull request merged**